playbooks/templates/k8s/namespace/01_matomo-config-map.yml.j2

```
@@ -238,6 +238,14 @@ data:
     # Copy config.ini.php.orig to matomo config folder
     cp /tmp/config.ini.php.orig /html/config/config.ini.php
 
+    # Copy php-fpm config files to emptydir
+    cp -a /opt/bitnami/php/etc/php-fpm.d/* /php-fpm-config/
+
+    # Update php-fpm config if needed
+    if [ "$0" == "matomo-ingestion" ]; then
```

Sicuro che $0 sia matomo-ingestion e non ./entrypoint.sh ?

---

Anche io pensavo così, ma non è `bash` che esegue il comando. Si tratta di `dash`, che quando è usata con -c, imposta `$0` con la stringa dopo il comando specificato in `-c`.
Maggiori dettagli in https://unix.stackexchange.com/questions/152391/bash-c-with-positional-parameters.

È anche necessario sostituire il `==` con `=` per funzionare correttamente con `dash`.
```suggestion
    if [ "$0" = "matomo-ingestion" ]; then
```